grommet/grommet
# Refractor Button.js to use hooks

What does this PR do?

This PR refactors Button.js to use hooks.
Where should the reviewer start?

Button/Button.js
What testing has been done on this PR?
storybook examples of buttons 
How should this be manually tested?
storybook
On a local project or with storybook.
Any background context you want to provide?

N/A
What are the relevant issues?

N/A
Screenshots (if appropriate)
Do the grommet docs need to be updated?
No
Should this PR be mentioned in the release notes?
No
Is this change backwards compatible or is it a breaking change?
backwards compatible
